tox.ini

```diff
@@ -4,12 +4,13 @@ python =
     3.8: py38
     3.9: py39
     3.10: py310
+    3.11: py311
 
 [tox]
 envlist =
     flake8,
-    py{37,38,39,310}-drf3-django{22,32}
-    py{38,39,310}-drf3-django{40}
+    py{37,38,39,310,311}-drf3-django{22,32}
```

Comment:
2.2 and 3.2 doesn't support Python 3.11 https://docs.djangoproject.com/en/4.1/faq/install/#what-python-version-can-i-use-with-django

Reply:
You're right -- sorry about that.

It is fixed now.

Reply:
`py37-drf3-django22` and `py37-drf3-django32` still fail with the following stack trace though:

```
ModuleNotFoundError: No module named 'distutils.cmd'
```

It does seem unrelated to this PR, so I have opened a separate issue for it: #187